modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/MiniPlayer.kt

```diff
@@ -78,13 +78,13 @@ class MiniPlayer @JvmOverloads constructor(context: Context, attrs: AttributeSet
         (binding.miniPlayerCardView.layoutParams as MarginLayoutParams).setMargins(margin, 0, margin, margin)
 
         val playBackground = if (FeatureFlag.isEnabled(Feature.MINI_PLAYER_DESIGN)) {
-            R.drawable.mini_player_play_background_32
+            R.drawable.mini_player_play_background_40
```

Comment:
Did you keep `if/else` in case we want to change it again?

Reply:
That's right. I'll remove it once it is in production and final.